src/rpc/misc.cpp

```diff
@@ -131,6 +131,9 @@ static RPCHelpMan createmultisig()
         if (!ParseOutputType(request.params[2].get_str(), output_type)) {
             throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, strprintf("Unknown address type '%s'", request.params[2].get_str()));
         }
+        if (output_type == OutputType::BECH32M) {
+            throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Bech32m multisig addresses cannot be created yet");
```

Comment:
Do we intend to ever create those through this RPC? I think you'd use `deriveaddress` or so instead.

Reply:
Probably not. Changed the message.